app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3:
+        if command[0] == "cp" and command[1] != command[2]:
+            with open(command[1], "r") as file_in, \
+                    open(command[2], "w") as file_out:
+                content_file = file_in.read()
+                file_out.write(content_file)
```

Comment:
1
what are command[1], command[2]?
create separate variables for them and call on demand
2
use parentheses instead of backslash to wrap the line
3
unite 7-8 lines into one line